javalib/src/main/scala/java/util/concurrent/ThreadLocalRandom.scala

```diff
@@ -22,98 +21,6 @@ class ThreadLocalRandom extends Random {
 
     super.setSeed(seed)
   }
-
-  def nextInt(least: Int, bound: Int): Int = {
```

Comment:
https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/util/concurrent/ThreadLocalRandom.html

suggests that a lot of overrides here need to stay. Removal intended?

Reply:
Likewise, `ThreadLocalRandom` is final (in the JavaDoc, but I now notice not in our code; so I'll add that). So if we can't behaviorally observe the difference, it's fine for us to have more or fewer overrides than what the JDK says.

For `ThreadLocalRandom` it's particularly fine, because it cannot be explicitly seeded. So no one can ever rely on a particular stream of outputs (as long as what we produce is at least as good, statistically speaking).